Makefile

```diff
@@ -164,8 +164,7 @@ docker buildx create --driver docker-container --name local --buildkitd-flags '-
 
 If you already have a compatible builder instance, you may use that instead.
 
-> Note: The security.insecure entitlement is only required, and used by the unit-tests target and targets which build container images
-for applications using `img` tool.
+> Note: The security.insecure entitlement is only required, and used by the unit-tests target.
```

Comment:
doesn;t generation of images still need that, or is this updated because it's now a `docker run` command?

Reply:
nope, it was actually never needed for a long time. we previously used `img` inside buildkit to build images of containers of services on talos rootfs, then it was refactored, then we dropped containers, so it works without it ;)